study.md

```diff
@@ -62,21 +73,43 @@ delicious apple.
 Copy the JSON that you wrote for the previous question and include the new information about the kitchen.
 
 ```json
-<!-- your answer here -->
+
+{ "kitchen" : {
+  "shelving" : "cupboard" {
+      "tableware" : ["bowls", "plates"]
+    },
+  "cookingDevice" : "electric stove" {
+    "stoveHeatingElements" : ["small", "medium", "large", "extra large"]
+  },
+  "spiceRackOne" : ["chives", "lemon grass", "mint"],
+  "spiceRackTwo" : ["cinnamon", "nutmeg", "vanilla"]
+},
+  "fruitBasket" : {
+    "Apples" : {
+      "Fuji" : "2",
+      "Gala" : "7",
+      "Golden delicious" : "1"
+    }
+  }
+}ç
 ```
 
 ## JSON Methods
 
 Can you write a method in the JSON that you wrote above?
 
 ```json
-<!-- your answer here -->
+JSON.stringify("kitchen")
+
+I think it would need to have a variable assignment such as let kitchen = {} for that method to work.
+I'm also a bit confused about the question because it makes me think I'm being asked to write a method similar to how we wrote JS methods, which I don't think is possible?
```

Comment:
Yup, that you're on to the idea behind this question. JSON is simply used as a way to represent data, and you cannot write a method in JSON